src/network/networkd-manager.h

```diff
@@ -115,6 +115,7 @@ struct Manager {
 
         /* sysctl */
         int ip_forwarding[2];
+        Hashmap *sysctls;
```

Comment:
So I very sure this should not be a global hashmap, but one specific for each link. i.e. we want to keep each of these hashmaps separate so that the moment a link goes away we can flush out all syssctl info for that link simply be freeing its link-specific hashmap

Reply:
There is a hashmap per link, and one global one for global sysctls `/proc/sys/net/ipv{4,6}/conf/{all,default}`

Reply:
Sorry, that was a first implementation, but then checking for changes was more complex because of the multiple maps.
I switched to a single map. If a link goes away we have some stale keys, but since the interface does not exist, noone can write into it.
Is it a problem, if an interface disappears and come back but unmanaged?